src/components/MemeCardComponent.js

```diff
@@ -17,17 +20,17 @@ export default function MemeCard(props) {
         } else if (voteDirection === 'down') {
             setVoteAction.payload = { downvotes: downvotes + 1 }
         }
-        console.log('vote action: ', setVoteAction)
 
         dispatch(setVoteAction);
```

Comment:
actions should be in other file dedicated to reducer. 
In component should be only something like this `dispatch(setVoteAction(voteDirection))`